contracts/upgradeability/OwnedUpgradeabilityProxy.sol

```diff
@@ -43,6 +46,13 @@ contract OwnedUpgradeabilityProxy is UpgradeabilityProxy {
     }
   }
 
+  /**
+   * @return the address of the implementation
+   */
+  function implementation() public view returns (address) {
+    return _implementation();
+  }
+
   /**
    * @dev Sets the address of the owner
    */
```

Comment:
NIT: L59, `setUpgradeabilityOwner` -> `_setUpgradeabilityOwner`? See also call in L24.